lib/controllers/widget/hardware_os_card.dart

```diff
@@ -0,0 +1,405 @@
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'OS version',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.osVersion ?? '-',
+                                style: TextStyle(
+                                    fontSize: 16, color: Colors.black54),
+                                overflow: TextOverflow.clip,
+                              )
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'Plataform Version',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.platformVersion ?? '-',
+                                style: TextStyle(
+                                    fontSize: 16, color: Colors.black54),
+                                overflow: TextOverflow.clip,
+                              )
+                            ],
+                          )),
+                      Padding(
+                          padding: EdgeInsets.only(top: 10, bottom: 10),
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            children: <Widget>[
+                              Text(
+                                'Device ID',
+                                style: TextStyle(
+                                    fontSize: 18,
+                                    color: Colors.black87,
+                                    fontWeight: FontWeight.w500),
+                                overflow: TextOverflow.clip,
+                              ),
+                              Text(
+                                widget._device.deviceId ?? '-',
+                                style: TextStyle(
```

Comment:
Same for this one

Reply:
I refactored the UI in another pull request.